src/app/api.py

```diff
@@ -6,6 +11,31 @@
 from src.app.monitor import api_router as monitor_router
 from src.jbi.router import api_router as jbi_router
 
+settings = get_settings()
+
+logging_config = {
+    "version": 1,
+    "formatters": {
+        "json": {
+            "()": "dockerflow.logging.JsonLogFormatter",
+            "logger_name": "jbi",
+        },
+    },
+    "handlers": {
+        "console": {
+            "level": settings.log_level.upper(),
+            "class": "logging.StreamHandler",
+            "formatter": "json",
+            "stream": sys.stdout,
+        }
+    },
+    "loggers": {
+        "request.summary": {"handlers": ["console"], "level": "INFO"},
+    },
+}
```

Comment:
Should this be part of the settings object instead? I've never used pydantic before

Reply:
I don't believe this needs to be part of the settings object/pydantic; it could be but it might be better in it's own `log.py` module...

However, I do think this can be handled on the app-startup:
```
@app.on_event("startup")
def startup_event():
    configure_logging()
```

then we can call `logging.getLogger("abc.xyz")` to get the logger with the correct configuration as well as "path" _(not sure what the name of the "abc.xyz" or "request.summary" would be called here, I'm calling this "path")_

Reply:
_Hmm, if this "startup" event's is used; this might come in handy: https://fastapi.tiangolo.com/advanced/testing-events/_